playbooks/setup_execution_and_consensus_full.yml

```diff
@@ -2,11 +2,14 @@
 # setups up the firewall rules for the machine
 # - import_playbook: tasks/start_firewall.yml
 
+# create user
+- import_playbook: tasks/create_bootstrap_user_and_provision_machine.yml
```

Comment:
This playbook is meant to be run against provisioned machines, so this shouldn't run